dev/traffic_router/conf/startup.properties

```diff
@@ -1,30 +0,0 @@
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-# Any environment variables you define here will become part of the startup
-# environment for the 'traffic_router' service. It is a good place to set
-# command line arguments for the Java command line or any environment specific
-# setting you want available to the traffic_router process.
-# This file is not replaced by the update process when a new version of traffic
-# router is installed.
-CATALINA_OPTS="\
-	-server -Xms2g -Xmx8g \
-	-Dlog4j.configuration=file:///opt/tomcat/conf/log4j2.xml \
-	-Djava.library.path=/usr/lib \
-	-Dorg.apache.catalina.connector.Response.ENFORCE_ENCODING_IN_GET_WRITER=false \
-	-XX:+UseG1GC \
-	-XX:+UnlockExperimentalVMOptions \
-	-XX:InitiatingHeapOccupancyPercent=30"
-JAVA_OPTS="\
-	-Djava.awt.headless=true \
-	-Djava.security.egd=file:/dev/./urandom"
```

Comment:
Are `CATALINA_OPTS` and `JAVA_OPTS` still set somewhere?

Reply:
They aren't but I don't consider that an issue since TR seems to be working just fine without it.